dropshot/src/api_description.rs

```diff
@@ -1241,6 +1241,29 @@ mod test {
         );
     }
 
+    #[should_panic(expected = "route paths must begin with a '/': 'I don't \
+                               start with a slash'")]
+    #[test]
+    fn test_badpath4() {
+        #[endpoint {
+            method = PUT,
+            path = "I don't start with a slash"
+        }]
+        async fn test_twobodies_handler(
+            _: Arc<RequestContext<()>>,
+        ) -> Result<Response<Body>, HttpError> {
+            unimplemented!();
+        }
+
+        let mut api = ApiDescription::new();
+        let error = api.register(test_twobodies_handler).unwrap_err();
+        assert_eq!(
+            error,
+            "only one body extractor can be used in a handler (this function \
+             has 2)"
```

Comment:
Is this assertion dead code? The function doesn't seem to have any body extractors, right?

Reply:
more copy pasta--sorry about that.